src/routes/_Navbar.svelte

```diff
@@ -30,7 +29,7 @@
     await store.signOutUser();
     browser.reload();
   };
-  
+
   const toggleNavIcon = () => {
     ariaExpanded = !ariaExpanded;
     ariaHidden = !ariaHidden;
```

Comment:
Calling `toggleDropdown()` in toggleNavIcon fixes this without having to pass down another similar prop to _Dropdown

Reply:
The issue is that we are using media query to display the dropdown. How would we know the condition for toggleDropdown in mount given this?